src/js/pages/Values/ValuesList.jsx

```diff
@@ -130,6 +131,7 @@ export default class ValuesList extends Component {
       }
     });
 
+    // Cordova Note Jan 26, 2022: The <PageContentContainer> wrapper replaced a few days ago with an empty <ValuesListWrapper> which will probably cause problems.
```

Comment:
PageContentContainer was removed from src/js/pages/Values/ValuesList.jsx, since it is not on its own page. It is only included on WebApp/src/js/pages/Values/VoterGuidesUnderOneValue.jsx, which has its own "PageContentContainer" wrapper. src/js/pages/Values/ValuesList.jsx should probably be moved into the Values component folder.

Reply:
Ok